### ISSUE TITLE
Does apply_force work when used outside a ForceGenerator?

### ISSUE THREAD

**Issue by jnferner** [rustsim/nphysics#179]
As per https://github.com/rustsim/nphysics/commit/c52d6eb4dd8a740c7b4dc017bfc3143b276554ce, I thought applying a force directly on a body works in the current version of nphysics. Indeed, if the `ForceType` is specified as `ForceType::VelocityChange`, the body moves, However I wasn't able to make anything move when using `ForceType::Force`. Am I missing something? 

I construct my body with the following code.

```rust
let polygon = ShapeHandle::new(
    ConvexPolygon::try_new(vec![
        Point::new(1.0, 2.0),
        Point::new(2.0, 2.0),
        Point::new(2.0, 3.0),
        Point::new(1.0, 3.0),
    ])
    .unwrap(),
);
let local_inertia = polygon.inertia(0.1);
let local_center_of_mass = polygon.center_of_mass();
RigidBodyDesc::new()
    .position(Isometry::new(Vector::new(2.0, 10.0), 3.0))
    .local_inertia(local_inertia)
    .local_center_of_mass(local_center_of_mass)
    .mass(100.0)
    .build(&mut world);
```
And I apply the force like this.
```rust
let force = Force::from_slice(&[5.00, 2.00, 0.5]);
body.apply_force(body.part_handle().1, &force, ForceType::Force, true);
```

Full code: https://github.com/jnferner/nphysics-playground/blob/master/src/bin/rigid_body.rs

I am using the following crate versions:
- nalgebra: 0.17.0
- ncollide2d: 0.18.1
- nphysics2d: 0.10.1

**Comment by sebcrozet** [comment]
Hi! Thank you for reporting this bug. This happens because the inertia of 2D rigid bodies is not correctly initialized when it it set manually (instead of automatically computed by the ColliderDesc).

This does not happen in 3D because the inertia of 3D rigid bodies are updated at each timestep.

This will be fixed by #180.

**Comment by jnferner** [comment]
Thanks for the quick fix :)
